### PR TITLE
expose createEmptyPolygon, reverse, makeValid

### PR DESCRIPTION
**Pull request by jw3126**

### CODE REVIEW

**Comment by jw3126** [comment]
@visr can you review? The PR is straight forward, it just exposes a couple of more functions and fixes a bug with `lineMerge`.

**Review by yeesian — APPROVED** [review]
It's great, thank you so much! Sorry it took me so long to review

File: Project.toml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 name = "LibGEOS"
 uuid = "a90b1aa1-3769-5649-ba7e-abc5a9d163eb"
 license = "MIT"
-version = "0.8.1"
+version = "0.8.2"
```
Comment (jw3126) [inline Project.toml:4]: @yeesian would it be possible to tag a release that includes this PR? That would be awesome, I already bumped the version.

Comment (visr) [reply Project.toml:4]: Thanks for the PR! I've added you as a contributor, so you can also tag releases :)

Comment (jw3126) [reply Project.toml:4]: Thanks a lot for adding me, I appreciate it!

**Pull request merged**